modules/doctor/fix16961.go

```diff
@@ -54,14 +55,14 @@ func fixUnitConfig16961(bs []byte, cfg *repo_model.UnitConfig) (fixed bool, err
 func fixExternalWikiConfig16961(bs []byte, cfg *repo_model.ExternalWikiConfig) (fixed bool, err error) {
 	err = json.UnmarshalHandleDoubleEncode(bs, &cfg)
 	if err == nil {
-		return
+		return false, nil
 	}
 
 	if len(bs) < 3 {
-		return
+		return false, err
 	}
 	if bs[0] != '&' || bs[1] != '{' || bs[len(bs)-1] != '}' {
-		return
+		return false, err
```

Comment:
In theory it's wrong to return `err` here because that error is not really the reason for the return.

Reply:
well I just kept the current behaviour

Reply:
well should we change?

Reply:
No, just wanted to mention it.